### PR TITLE
[ci-visibility] Small fixes in javascript

### PR DESCRIPTION
**Pull request by juan-fernandez**
### What does this PR do?
* Fix note about test modules in javascript.

### Preview

https://docs-staging.datadoghq.com/juan-fernandez/small-fix-js/continuous_integration/tests/#module

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/continuous_integration/tests/javascript.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ Supported test frameworks:
 The instrumentation works at runtime, so any transpilers such as TypeScript, Webpack, Babel, or others are supported out of the box.
 
 ### Test suite level visibility compatibility
-[Test suite level visibility][5] is fully supported from `dd-trace>=3.14.0` and `dd-trace>=2.27.0`. Jest, Mocha, Playwright, Cypress, and Cucumber are supported.
+[Test suite level visibility][5] is fully supported from `dd-trace>=3.14.0` and `dd-trace>=2.27.0`. Jest, Mocha, Playwright, Cypress and Cucumber are supported.
```
Comment (drichards-87) [inline content/en/continuous_integration/tests/javascript.md:44]:
```suggestion
[Test suite level visibility][5] is fully supported from `dd-trace>=3.14.0` and `dd-trace>=2.27.0`. Jest, Mocha, Playwright, Cypress, and Cucumber are supported.
```

Comment (drichards-87) [reply content/en/continuous_integration/tests/javascript.md:44]: Can we leave the comma here? It's actually part of our style guide to use the Oxford comma, so for consistency we prefer to keep it.

File: content/en/continuous_integration/tests/_index.md [hunk 2]
```diff
@@ -103,7 +103,7 @@ The definition of module changes slightly per language:
 
 * In .NET a test module groups every test that is run under the same [unit test project][8].
 * In Swift a test module groups every test that is run for a given bundle.
-* In JavaScript there are no test modules.
+* In JavaScript the test modules map one to one to test sessions.
```
Comment (drichards-87) [inline content/en/continuous_integration/tests/_index.md:106]:
```suggestion
* In JavaScript the test modules map one-to-one to test sessions.
```

**Review by drichards-87 — CHANGES_REQUESTED** [review]
Left you a couple of minor pieces of feedback.

File: content/en/continuous_integration/tests/javascript.md (hunk 1 above)
Comment (juan-fernandez) [reply content/en/continuous_integration/tests/javascript.md:44]: oh sure, I didn't know that. Thanks

**Review by drichards-87 — APPROVED** [review]
Looks good. I'll get this merged now.

**Pull request merged**